### ISSUE TITLE
rl-tcp example ran once only

### ISSUE THREAD

**Issue by sanjayjaiman** [hust-diangroup/ns3-ai#38]
I ran the "rl-tcp" example once by following the instructions - 
Window 1:  ./waf --run "rl-tcp"
Window 2: python testtcp.py (in the "contrib/example/rl-tcp" dir)

First time it seemed to run fine to completion.
I tried it again and "python testtcp.py" script exits without any I/O on either window.
I saw there were quite a few processes with me as owner running (possibly due to previous ns-3 runs).  I killed them all and still the same problem.
"netstat -an | grep 5000"   does not yield anything

**Comment by Mauriyin** [comment]
Could you check whether the shared memory successfully released? You can run free.sh to release the memory. Then you can try to re-run the example. 

Also, you can run [this](https://github.com/hust-diangroup/ns3-ai/blob/master/example/rl-tcp/run_rl_tcp.py) new script to directly test the rl-tcp, no need to run it in two separate windows.

**Comment by sanjayjaiman** [comment]
Thanks Mauriyin,  Here was my experience: With the system in the same state as before -

1. I ran "run_rl_tcp.py" and that behaved same as "testtcp.py" - it simply exited
2. I ran "./freeshm.sh " in the "ns3-ai" dir.  That seemed to free up memory.
3. After that "rl-tcp" and "testtcp.py" ran fine.
4. Running it again, I ran into the same issue again.  After freeing up shared memory, it ran fine.
5. Seems like I have to run "./freeshm.sh" after every run.  Is this another issue I should open or this one is good enough?

Thanks,
Sanjay

**Comment by Mauriyin** [comment]
Yes. It's fine you can free the memory before each running. We are looking at these issues. Thanks.
